common-lib/src/main/scala/com/gu/workflow/api/ApiUtils.scala

```diff
@@ -68,8 +68,8 @@ object ApiUtils {
       case JsSuccess(a, _) => ApiResponseFt.Right(a)
       case error@JsError(_) =>
         val errMsg = errorMsgs(error)
-        Logger.error(s"JsonParseError failed to parse the json. Error(s): $errMsg 400 badrequest")
-        ApiResponseFt.Left(ApiError("JsonParseError", s"failed to parse the json. Error(s): $errMsg", 400, "badrequest"))
+        Logger.error(s"JsonParseError failed to parse the json. Error(s): $errMsg 400 badrequest. Json: $jsValue")
```

Comment:
I don't know what the json response would be but is there going to be anything sensitive about our users that we wouldn't want to log? Though we clear out logs relatively quickly so I doubt it would be an issue